src/stories/Forms/Select/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+import { GlobalInputProps } from '../../../shared/types';
+
+interface WrapperProps {
+  hasError?: boolean;
+}
+
+const Wrapper = styled.div<WrapperProps>`
+  select {
+    display: block;
+    width: 100%;
+    height: var(--spacing-48);
+    padding: 0 var(--spacing-16);
+    border: ${(p) =>
+      p.hasError ? 'var(--border-error)' : 'var(--border-primary)'};
```

Comment:
Also worried about the border size here?

Reply:
Will move the padding into the element if it's not already.